### PR TITLE
Created a readme file

### PR DESCRIPTION
**Pull request by AvantikaSharma2307**
**Readme** **created**
created a readme file for the website.
Contains Intro part, how to install, what need it cater etc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+### Table of Contents
+* [Introduction](#introduction)
+* [Features](#features)
+* [Installation](#installation)
+* [Usage](#usage)
+* [Contributing](#contribution)
+<!-- License -->
+---
+## Introduction  <a name="introduction"></a>
+The CNCG Delhi website aims to serve as a central hub for members and enthusiasts of the computer and networking community in Delhi. Through this website, we aim to provide updates on events, workshops, and other relevant activities organized by CNCG Delhi.
```
Comment (satyampsoni) [inline README.md]:
```suggestion
We are the New Delhi chapter of the Cloud Native Community Group (CNCG), operating under the [Cloud Native Computing Foundation (CNCF).](http://cncf.io/) Our focus is on knowledge sharing and exploring the potential of cloud native technologies. Through our meetups, educational sessions, and practical activities, we aim to empower individuals in navigating the cloud native ecosystem. We are dedicated to hosting at least one meetup every month and take pride in being recognized as the leading CNCG community worldwide. We prioritize creating an inclusive learning environment, adhering to the [CNCF Code of Conduct](https://www.cncf.io/conduct/) to ensure everyone feels welcome and respected.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,35 @@
...
+
+## Features <a name="features"></a>
+* Event Calendar: Stay updated with upcoming events, workshops, and meetups organized by CNCG Delhi.
+* Resource Repository: Access resources, presentations, and materials shared during past events.
+* Community Forums: Engage with fellow members, ask questions, and participate in discussions related to computer and networking technologies.
+* Membership Registration: Join CNCG Delhi community by registering through the website.
+##  Installation <a name="installation"></a>
+To deploy the CNCG Delhi website locally, follow these steps:
```
Comment (satyampsoni) [inline README.md]:
```suggestion
To set up the CNCG Delhi website locally, following are the steps:
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,35 @@
...
+
+* Clone the repository: git clone https://github.com/cncgnd/website.git
```
Comment (satyampsoni) [inline README.md]:
```suggestion
* Clone the repository
`git clone https://github.com/cncgnd/website.git`
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,35 @@
...
+* Navigate to the project directory: cd website
```
Comment (satyampsoni) [inline README.md]:
```suggestion
* Navigate to the project directory: 
`cd website`
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,35 @@
...
+* Install dependencies: npm install
```
Comment (satyampsoni) [inline README.md]:
```suggestion
* Install dependencies: 
`npm install`
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,35 @@
...
+* Start the development server: npm run dev
```
Comment (satyampsoni) [inline README.md]:
```suggestion
* Start the development server: 
 `npm run dev`
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,35 @@
...
+## Usage <a name="usage"></a>
+The CNCG Delhi website is designed to be user-friendly. Simply navigate through the menu options to explore different sections of the website. You can register for membership, browse upcoming events, access resources, and engage with the community through forums.
+
+## Contributing <a name="contribution"></a>
+We welcome contributions from the community to improve the CNCG Delhi website. If you find any issues or have suggestions for enhancements, feel free to open an issue or submit a pull request. Please adhere to our code of conduct when contributing.
```
Comment (satyampsoni) [inline README.md]:
```suggestion
We welcome contributions from the community to improve the CNCG Delhi website. If you find any issues or have suggestions for enhancements, feel free to open an issue or submit a pull request. Please adhere to [CNCF code of conduct](https://www.cncf.io/conduct/) when contributing.
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,35 @@
-# website
-This repository belongs to CNCG New Delhi Portfolio website.
+
+#  CNCG Delhi Website                                                                                                                                        
+
+### Welcome to the CNCG Delhi website repository!
+ ##### This website serves as a platform to provide information about the activities, events, and initiatives undertaken by the Computer and Networking Community Group (CNCG) in Delhi.
```
Comment (satyampsoni) [inline README.md]:
```suggestion
 ##### This website serves as a platform to provide information about the activities, events, and initiatives undertaken by Cloud Native Community Groups (CNCG) in New Delhi.
```

**Comment by AvantikaSharma2307** [comment]
DONE

**Review by AvantikaSharma2307 — COMMENTED** [review]
done

**Review by satyampsoni — APPROVED** [review]

**Pull request merged**